bundles/org.eclipse.passage.lic.api/src/org/eclipse/passage/lic/internal/api/conditions/VersionMatch.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     ArSysOp - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.passage.lic.internal.api.conditions;
+
+/**
+ * 
+ * <p>
+ * Part of licensing {@linkplain Condition} that declares license demands
+ * regarding the feature version.
+ * </p>
+ * <p>
+ * This way or another a license defines which version of a feature is allowed
+ * to use. It also defines <i>how far</i> from this specified version can be the
+ * actual version of the feature that has been plugged to the product under
+ * licensing.
+ * </p>
+ * 
+ * @see Condition
+ */
+public interface VersionMatch {
```

Comment:
Need to revisit this to ensure rich version range support, most probably it is sufficient in this form

Reply:
It's quite general:  
 - data part supplies any string, as compicated as you like, and the
 - algorithmic part `MatchingRule` knows how to deal with it. This one, again, is general. It demands nothing from data, it just matches two strings according it's hidden logic.

I suppose this is enough. Currently I'm reworking `LicensingVersions` on these interfaces - it goes well.